app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -91,6 +91,17 @@ def link_to_field(name, value, label = nil, facet_hash = {})
       link_to(label, main_app.search_catalog_path(state))
     end
 
+    def index_facet_link options={}
```

Comment:
@jcoyne: @mtribone and I were both wondering why you had recommended not using that routine for facets on slack last night, but since that was your recommendation we worked most of the day to not use that method.